### PR TITLE
Draft of casc j9 scripts

### PR DESCRIPTION
**Pull request by ajreynol**
Based on recent StarExec runs, modified from casc26 scripts.

### CODE REVIEW

File: contrib/run-script-cascj9-fnt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+here=`dirname $0`
+cvc4=$here/cvc4
+bench="$1"
+
+file=${bench##*/}
+filename=${file%.*}
+
+echo "------- cvc4-fnt casc 26 : $bench at $2..."
```
Comment (4tXJ7f) [inline contrib/run-script-cascj9-fnt]: I guess this should say `casc j9`.

File: contrib/run-script-cascj9-fof [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# script is used for FOF division, also SLD division
+
+here=`dirname $0`
+cvc4=$here/cvc4
+bench="$1"
+
+file=${bench##*/}
+filename=${file%.*}
+
+echo "------- cvc4-fof casc 26 : $bench at $2..."
```
Comment (4tXJ7f) [inline contrib/run-script-cascj9-fof]: I guess this should say `casc j9`.

File: contrib/run-script-cascj9-tfa [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+here=`dirname $0`
+cvc4=$here/cvc4
+bench="$1"
+
+file=${bench##*/}
+filename=${file%.*}
+
+echo "------- cvc4-tfa casc 26 : $bench at $2..."
```
Comment (4tXJ7f) [inline contrib/run-script-cascj9-tfa]: I guess this should say `casc j9`.

File: contrib/run-script-cascj9-tfa [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+# use: trywith [params..]
+# to attempt a run.  If an SZS ontology result is printed, then
+# the run script terminates immediately.  Otherwise, this
+# function returns normally.
+function trywith {
+  limit=$1; shift;
+  echo "--- Run $@ at $limit...";
+  (ulimit -S -t "$limit";$cvc4 --lang=tptp --no-checking --no-interactive --force-logic="UFNIRA" --dump-instantiations --inst-format=szs --force-no-limit-cpu-while-dump "$@" $bench) 2>/dev/null |
```
Comment (4tXJ7f) [inline contrib/run-script-cascj9-tfa]: Naive question: why is there no `--force-logic` in the other scripts?

Comment (ajreynol) [reply contrib/run-script-cascj9-tfa]: FOF and FNT are on problems with SMTLIB logic UF, which is the default logic of the tptp parser. Whereas TFA "type first-order with arithmetic" is UFNIRA.

**Pull request merged**